docs/en/get-started/layered-web-application.md

```diff
@@ -105,6 +105,10 @@ Here, you can select the database management systems (DBMS){{ if DB == "EF" }} a
 
 If you uncheck the *Kubernetes Configuration* option, the solution will not include the Kubernetes configuration files, such as Helm charts and other Kubernetes-related files. You can also specify *Social Logins*; if you uncheck this option, the solution will not be configured for social login. Lastly, you can specify the *Include Tests* option to include or exclude the test projects from the solution.
 
+On the next screen, you can configure the modularity options for your solution:
+
+![abp-studio-new-solution-modularity](images/abp-studio-new-solution-dialog-modularity.png)
+
```

Comment:
If you select the _Setup as a modular solution_ option, the solution is created more ready for [modular monolith development](../tutorials/modular-crm/index.md) and allows you to add sub-modules during the solution creation phase.